include/deal.II/grid/tria.h

```diff
@@ -550,12 +550,12 @@ namespace internal
  * associated with curves in 2d or surfaces in 3d) are the ones one wants to
  * use in the boundary element method.
  *
- * The name of the class is mostly hierarchical and is not meant to imply that
+ * The name of the class is mostly historical and is not meant to imply that
```

Comment:
That was an interesting statement before the change :rofl:

Reply:
Yes. The longer I'm in the US, I find that I make the same mistakes as native speakers in that I accidentally write words that sounds similar but are different (in particular there/they're/their, but this is probably another example of the same problem).

This was introduced in #10979. Both @tjhei and @masterleinad missed it too :-)